core/src/main/kotlin/org/evomaster/core/problem/graphql/GqlConst.kt

```diff
@@ -0,0 +1,46 @@
+package org.evomaster.core.problem.graphql
+
+object GqlConst {
+
+    /**
+     * This tag is for the GQL union type. Needed in getValueAsPrintableString to print out things like:
+     *   fieldXName{
+     *        ... on UnionObject1 {
+     *           field
+     *        }
+     *        ... on UnionObjectN {
+     *          field
+     *        }
+     *   }
+     */
+    const val UNION_TAG = "#UNION#"
+    /**
+     * Those tags are for the GQL interface type. Needed in getValueAsPrintableString to print out things like:
+     *     fieldXName{
+     *        field1
+     *        fieldN
+     *        ... on InterfaceObject1 {
+     *           field
+     *        }
+     *        ... on InterfaceObjectN {
+     *          field
+     *        }
+     *     }
+     */
+    const val INTERFACE_BASE_TAG = "#BASE#"
+    const val INTERFACE_TAG = "#INTERFACE#"
```

Comment:
what is the difference between `INTERFACE_BASE_TAG` and `INTERFACE_TAG`?

Reply:
eg:
let s take this schema:
```
interface Book {
  title: String!
}

type Textbook implements Book {
  title: String!
  courses: String!
}

type ColoringBook implements Book {
  title: String!
  colors: [String!]!
}

type Query {
  books: [Book!]!
}
```
And this Query:
```
{
  books {
    title
    ... on Textbook {
      courses # Only present in ColoringBook
    }
    ... on ColoringBook {
      colors # Only present in ColoringBook
    }
            }
}
```
INTERFACE_BASE_TAG: is used to annotate the interface type Book (so we can construct all the interface fields, like: title ) and
INTERFACE_TAG: is used to annotate the query field name books (which has the type: interface) (so we can construct ... on Textbook and ... on ColoringBook)

Reply:
add this explanations as comments

Reply:
Done